### PR TITLE
[FLINK-3189] Error while parsing job arguments passed by CLI

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by StephanEwen** [comment]
Can you add a test case that guards this change?

**Comment by mjsax** [comment]
Sure. Will do that.

**Comment by mjsax** [comment]
Done. Failing Travis is unrelated.

File: flink-clients/src/test/java/org/apache/flink/client/CliFrontendInfoTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void testShowExecutionPlan() {
 		replaceStdOut();
 		try {
 
-			String[] parameters = new String[] { CliFrontendTestUtils.getTestJarPath() };
+			String[] parameters = new String[] { CliFrontendTestUtils.getTestJarPath(), "-v", "true", "/dev/null", "/dev/null"};
```
Comment (fhueske) [inline flink-clients/src/test/java/org/apache/flink/client/CliFrontendInfoTest.java]: `-v` is a valid option. I would change it to `-f` which is also the option initially causing the problem.

**Comment by fhueske** [comment]
Will change the option to `-f` and merge this PR

**Pull request closed without merging**